ByBit.Net/Objects/Models/Socket/Spot/BybitSpotStopOrderUpdate.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Bybit.Net.Converters;
+using Bybit.Net.Enums;
+using CryptoExchange.Net.Converters;
+using Newtonsoft.Json;
+
+namespace Bybit.Net.Objects.Models.Socket.Spot
+{
+    /// <summary>
+    /// Spot order update
+    /// </summary>
+    public class BybitSpotStopOrderUpdate : BybitSocketEvent
+    {
+        /// <summary>
+        /// Symbol
+        /// </summary>
+        [JsonProperty("s")]
+        public string Symbol { get; set; } = string.Empty;
+        /// <summary>
+        /// Client order id
+        /// </summary>
+        [JsonProperty("c")]
+        public string ClientOrderId { get; set; } = string.Empty;
+        /// <summary>
+        /// Order side
+        /// </summary>
+        [JsonProperty("S"), JsonConverter(typeof(OrderSideConverter))]
+        public OrderSide Side { get; set; }
+        /// <summary>
+        /// Order type
+        /// </summary>
+        [JsonProperty("o"), JsonConverter(typeof(OrderTypeSpotConverter))]
+        public OrderType Type { get; set; }
+        /// <summary>
+        /// Time in force
+        /// </summary>
+        [JsonProperty("f"), JsonConverter(typeof(TimeInForceSpotConverter))]
+        public TimeInForce TimeInForce { get; set; }
+        /// <summary>
+        /// Quantity
+        /// </summary>
+        [JsonProperty("q")]
+        public decimal Quantity { get; set; }
+        /// <summary>
+        /// Price
+        /// </summary>
+        [JsonProperty("p")]
+        public decimal Price { get; set; }
+        /// <summary>
+        /// Status
+        /// </summary>
+        [JsonProperty("X"), JsonConverter(typeof(OrderStopStatusSpotConverter))]
+        public OrderStatus Status { get; set; }
+        /// <summary>
+        /// Order id
+        /// </summary>
+        [JsonProperty("i")]
+        public string OrderId { get; set; }
```

Comment:
Could you have a look at the warnings? In these cases, just add a `= string.Empty;` after the properties